uelc/main/models.py

```diff
@@ -581,6 +581,13 @@ def display_title(self):
     def display_description(self):
         return self.description
 
+    @classmethod
+    def add_form(self):
```

Comment:
Nik, good point out. I have been confused at times what the scope of "self" was when debugging. When initially writing this I was using the old quizblock as a guide which was had self. Changing it over to cls is a good idea.